cockroachcloud/cmek-ops-aws.md

```diff
@@ -82,7 +82,7 @@ After you have provisioned the cross-account IAM role and CMEK key for your Cock
 
 ### Option B: Use the Vault AWS-KMS secrets engine to create the CMEK key
 
-#### Prerequisites
+#### Before you begin
 
 - [Vault enterprise license](https://www.vaultproject.io/docs/enterprise)
```

Comment:
Under the new title, these two items likely need a little more context. As prereqs: a simple list suffices: "procure each item on this list" is inferable.

Less so with "before you begin". What do you think of:

> You must have a [Vault enterprise license].
> You must have [installed Vault enterprise edition locally].